android/libraries/rib-android/src/main/java/com/uber/rib/core/RibActivity.kt

```diff
@@ -6,9 +6,23 @@ import android.support.v7.app.AppCompatActivity
 import android.view.ViewGroup
 import com.badoo.ribs.android.ActivityStarter
 import com.badoo.ribs.android.IntentCreator
+import com.badoo.ribs.android.PermissionRequester
+import com.badoo.ribs.android.PermissionRequesterImpl
+import com.badoo.ribs.core.Identifiable
 import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.requestcode.RequestCodeRegistry
+import io.reactivex.Observable
 
-abstract class RibActivity : AppCompatActivity(), ActivityStarter, IntentCreator {
+abstract class RibActivity : AppCompatActivity(),
+    ActivityStarter,
+    IntentCreator,
+    PermissionRequester {
+
+    private val requestCodeRegistry = RequestCodeRegistry()
+    private val permissionRequester = PermissionRequesterImpl(
```

Comment:
Code style question.
Why are you braking lines almost for every `function` or `constructor` even if those contain 2, 3 arguments and fits the line (less then 150 symbols)?

This is just one of the examples.

I can't find such rule/recommendation either on [Kotlin Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html) or [Android Kotlin Style Guide](https://developer.android.com/kotlin/style-guide).

cc @lukaville

Reply:
+1
And in case of line breaking and assignment operator, I think it would more readable if you break right after "=".
So either like this (if it fits one line):
`private val permissionRequester = PermissionRequesterImpl(this, requestCodeRegistry)`
or like this (if does not fit):
```
private val permissionRequester = 
    PermissionRequesterImpl(
        line1,
        line2,
        line3,
        line4,
    )
```

I think indentations are very important here, because last closing ")" belongs to PermissionRequesterImpl and not to "private val permissionRequester".

Reply:
Fixed.